stack/adding-a-new-package.rst

```diff
@@ -43,11 +51,33 @@ The roles assigned to these teams should typically be "Write" for "Data Manageme
   The automated builds use the team membership to determine the type of tag to be applied.
   Having the code reside in the ``lsst`` or ``lsst-dm`` organization on GitHub is not sufficient.
 
+repos.yaml
+----------
+
 The new package must be added to the `etc/repos.yaml file in the lsst/repos repository`_ along with its corresponding GitHub URL.
 This file is governed by a "self-merge" policy: upon opening a pull request, it will be checked by GitHub Actions, and developers may merge without further review on success.
-This change **must** be merged before the package can be built on Jenkins.
+This change **must** be merged before the package can be built on Jenkins, and this should be done prior to the RFC being filed.
 Refer to :jira:`RFC-75` for background.
 
+Package and pipelines docs
+--------------------------
+
+Documentation for the new package must be built and linked from the main `Science Pipelines docs`_ page.
+To make that documentation visible as part of your RFC before your package is included in the official builds:
+
+1. Follow the instructions for :ref:`adding a package to pipelines.lsst.io <add-to-pipelines-lsst-io>` on a ticket branch.
+2. :ref:`Build the Science PIpelines docs locally <local-pipelines-lsst-io-build>` on that branch.
+3. Copy the ``_build/html`` directory from your pipelines build to a place that's publicly viewable (e.g. your public web path on :doc:`the USDF </usdf/storage>`).
+4. Include a link to those built docs in your RFC.
```

Comment:
Again, I think this is premature.  The RFC happens at the beginning of the process, not the end.

Reply:
I've added some wording about RFC timing: please let me know what you think.